gst/nnstreamer/tensor_data.c

```diff
@@ -0,0 +1,296 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * @file	tensor_data.c
+ * @date	10 Mar 2021
+ * @brief	Internal functions to handle various tensor type and value.
+ * @see		http://github.com/nnstreamer/nnstreamer
```

Comment:
Mixed tabs and spaces in the indentation of this line.

Reply:
In code view, this is correct.
https://github.com/jaeyun-jung/nnstreamer/blob/0c3db1f73e0f5a56fd5085f0d4c0a9f2584656ea/gst/nnstreamer/tensor_data.c#L8

Reply:
@jaeyun-jung These are tabs. 

![image](https://user-images.githubusercontent.com/2772376/111959800-45c38800-8b32-11eb-955e-9794e6370275.png)

Reply:
changed to single tab for the tag `@see / @bug`